### ISSUE TITLE
Slightly weird validation curve

### ISSUE THREAD

**Issue by segermans** [ChuhuaW/SGNet.pytorch#14]
Hi guys, 

First of all, thanks for your great contribution! I am training the deterministic model on JAAD from scratch. I am getting results which are close to the published results, however the validation loss curve looks a bit strange; 

- First 5 - 7 epochs: the model is not improving on validation loss at all (even slightly losing performance) stagnating at around 3.3-3.5. - Epoch 7 - 9: sudden drop in validation loss to about 2.0.
- Epoch 9 - end (50): Validation loss creeps down to about 1.65. 

Did you guys also experience this stagnating loss curve at the start after which a sudden drop occured? And if so, do you have an explanation for this?

Thanks in advance for answering the question. Cheers, Seger

**Comment by ChuhuaW** [comment]
It is a really good question! Yes, we had the same experience on JAAD, but it is much smoother for PIE. And the learning rate is not decreased in the first few epochs. I don't have a very good explanation for it, but this [post ](https://www.reddit.com/r/MachineLearning/comments/ouiegi/d_sudden_drop_in_loss_after_hours_of_no/) may be related. I really like one of the explanation there: 

> If I had to hazard a guess as to why this happened theoretically is that the model was moving steadily across a plateau on the loss surface and then struck gold by finding a local minima leading to that drastic decrease in the loss.
